sats-dna/src/main/kotlin/com/sats/dna/components/SatsChallengeBackground.kt

```diff
@@ -27,25 +23,25 @@ fun SatsChallengeBackground(modifier: Modifier = Modifier, isEnabled: Boolean =
         color = SatsTheme.colors.backgrounds.fixed.primary.default.bg,
         useMaterial3 = true,
     ) {
-        Box {
-            Image(
-                imageVector = ImageVector.vectorResource(R.drawable.challenge_background_top),
-                contentDescription = null,
-                modifier = Modifier.fillMaxWidth(),
-                contentScale = ContentScale.FillBounds,
-                colorFilter = if (!isEnabled) {
-                    ColorFilter.colorMatrix(ColorMatrix().apply { setToSaturation(0f) })
-                } else {
-                    null
-                },
-            )
-
+        Box(
+            Modifier.challengeBackground(),
+        ) {
             content()
         }
     }
 }
 
-fun Modifier.greyScale() = this.then(GreyScaleModifier())
+private fun Modifier.greyScale() = this.then(GreyScaleModifier())
+
+@Composable
+private fun Modifier.challengeBackground(): Modifier =
```

Comment:
This could probably just be inlined, started with an approach where this modifier would be public and then it made more sense to do it like this 🙏

Reply:
Agree. We can do the same with the `Modifier.greyScale()` above it, too. 👍